packages/browser/src/helpers.ts

```diff
@@ -213,3 +213,40 @@ export function injectReportDialog(options: ReportDialogOptions = {}): void {
 
   (document.head || document.body).appendChild(script);
 }
+
+/**
+ * Convert a base64-encoded string back to plaintext. Does not include error handling.
+ *
+ * (This is here (and in the node package) rather than in @sentry/utils to prevent webpack from bundling the entire
+ * Buffer module when creating SDK users' vendor bundles for the browser. Used in @sentry/utils.string.)
```

Comment:
I'd avoid making certain cross-references that can get outdated. Comments are not good for that, and we have tools to figure out where a certain piece of code is used.
I'd also avoid putting such comments in the jsdoc of functions, since they are unrelated to how the function works and would end up polluting the generated documentation.

My suggestion is to have all related functions in a single file and have a single comment applying to the whole file.

File: `packages/browser/src/base64.ts`
```javascript
// This file is here instead of in @sentry/utils such that users of the Browser SDK using webpack
// do not have Node's Buffer module unnecessarily included in their bundles.

// ...
```

Reply:
I think you're right about not polluting the JSDoc with it, but I kept a similar comment because, sure, you can search for something across packages, but who thinks to do that if you're just changing one test for whatever reason? This way, anyone changing things in one place knows immediately that it needs to be done elsewhere as well (even if that change is taking them out of sync for some reason, in which case they'll know to delete the comment in both places!).